src/platyPS/platyPS.psm1

```diff
@@ -2910,15 +2910,55 @@ function ConvertPsObjectsToMamlModel
 
         $MamlExampleObject.Introduction = $Example.introduction
         $MamlExampleObject.Title = $Example.title
+        
+        # first line in multiline example can be found in $Example.code, second and other lines are in $Example.remarks
```

Comment:
Can you add a test for this change?

Reply:
Yes, test (and another small fix) added.